src/index.js

```diff
@@ -41,61 +42,82 @@ const MODES = {
   },
 };
 
-const args = process.argv.slice(2);
+const argv = parseArgs(process.argv);
```

Comment:
Vector? Would be my guess. If I'm truthful I named it this way because that's what the README did that I copied code from :P. I know, I'm a hack.